examples/react_webpack/BUILD.bazel

```diff
@@ -13,27 +13,29 @@ sass(
     data = ["styles.scss"],
 )
 
-tsc(
-    name = "compile",
-    outs = ["index.js"],
-    args = [
-        "$(execpath index.tsx)",
-        "$(execpath types.d.ts)",
-        "--outDir",
-        "$(RULEDIR)",
-        "--lib",
-        "es2015,dom",
-        "--jsx",
-        "react",
-    ],
-    data = [
+ts_project(
+    name = "tsconfig",
+    srcs = [
         "index.tsx",
         "types.d.ts",
+    ],
+    # The tsconfig.json file doesn't enable "declaration" or "composite"
+    # so tsc won't produce any .d.ts outputs. We need to tell Bazel not
+    # to expect those outputs being produced.
+    declaration = False,
+    deps = [
         "@npm//@types",
         "@npm//csstype",
     ],
 )
 
+# If you don't want to write //:tsconfig as the label for the TypeScript compilation,
+# use an alias like this, so you can write //:compile_ts instead.
+alias(
+    name = "compile_ts",
```

Comment:
nit: use tsconfig attribute and use "compile" as ts_project name to avoid alias?

Reply:
oops, this BUILD file should have been updated for the change of `declaration` default to False too. will boil it down to the minimum